trie/testutil/utils.go

```diff
@@ -59,3 +60,14 @@ func RandomNode() *trienode.Node {
 	val := RandBytes(100)
 	return trienode.New(crypto.Keccak256Hash(val), val)
 }
+
+// RandomString generates specified length random string.
+func RandomString() string {
```

Comment:
only use to UT? move it to UT file?

Reply:
The contents of this file is used for UT.